testsV2/ut_repy2api_copycontext.py

```diff
@@ -0,0 +1,10 @@
+"""
+Check that copy and repr of _context SafeDict don't result in inifinite loop
+"""
+#pragma repy
+
+# Create an almost shallow copy of _context
+# Contained self-reference is moved from _context to _context_copy
```

Comment:
There is a lot of useful info on the issue summary/replies that you should consider adding to these comments, one being that Python's repr() truncates part of the SafeDict.